corehq/apps/integration/payments/tests/test_views.py

```diff
@@ -162,6 +172,102 @@ def test_verify_rows(self):
         assert response.context['failure_count'] == 0
 
 
+@es_test(requires=[case_search_adapter], setup_class=True)
+class TestPaymentsVerifyTableFilterView(BaseTestPaymentsView):
+    urlname = PaymentsVerificationTableView.urlname
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.factory = CaseFactory(cls.domain)
+        cls.case_list = [
+            _create_case(
+                cls.factory,
+                name='foo',
+                data={
+                    PaymentProperties.BATCH_NUMBER: 'B001',
+                    PaymentProperties.PAYMENT_VERIFIED: True,
+                    PaymentProperties.PAYMENT_SUBMITTED: True,
+                }),
+            _create_case(
+                cls.factory,
+                name='bar',
+                data={
+                    PaymentProperties.BATCH_NUMBER: 'B001',
+                    PaymentProperties.PAYMENT_VERIFIED: True,
+                    PaymentProperties.PAYMENT_SUBMITTED: False,
+                }),
+            _create_case(
+                cls.factory,
+                name='baz',
+                data={
+                    PaymentProperties.BATCH_NUMBER: 'B001',
+                }),
+        ]
+        case_search_adapter.bulk_index(cls.case_list, refresh=True)
+
+    @classmethod
+    def tearDownClass(cls):
+        for case in cls.case_list:
+            case.delete()
+        super().tearDownClass()
+
+    @flag_enabled('MTN_MOBILE_WORKER_VERIFICATION')
```

Comment:
nit: I usually also add a test to check what would happen if feature is not enabled.